pkgs/by-name/de/decibels/package.nix

```
@@ -60,16 +55,18 @@ stdenv.mkDerivation rec {
   '';
 
   passthru = {
-    updateScript = nix-update-script { };
+    updateScript = gnome.updateScript {
+      packageName = "decibels";
+    };
   };
 
   meta = {
     description = "Play audio files";
-    homepage = "https://gitlab.gnome.org/GNOME/Incubator/decibels";
-    changelog = "https://gitlab.gnome.org/GNOME/decibels/-/blob/${version}/NEWS?ref_type=tags";
+    homepage = "https://gitlab.gnome.org/GNOME/decibels";
+    changelog = "https://gitlab.gnome.org/GNOME/decibels/-/blob/${finalAttrs.version}/NEWS?ref_type=tags";
     license = lib.licenses.gpl3Only;
-    teams = [ lib.teams.gnome-circle ];
+    teams = [ lib.teams.gnome lib.teams.gnome-circle ];
```

Formatter is complaining.
amend! decibels: Add GNOME team as maintainers
```suggestion
    teams = [
      lib.teams.gnome
      lib.teams.gnome-circle
    ];
```
I think this is the correct formatting according to the logs I read yesterday

---

I think a `with lib.teams;` would be good here for readability.

---

The simplest option is just to run `nix-shell --run treefmt`